spec/truefi2/BorrowingMutex.spec

```diff
@@ -1,11 +1,19 @@
 methods {
     isUnlocked(address) returns bool envfree
     isBanned(address) returns bool envfree
+    locker(address) returns address envfree
 }
 
 definition isReasonableAddress(address alice) returns bool = alice != 0 && alice != 1;
 definition isReasonableEnv(env e) returns bool = isReasonableAddress(e.msg.sender);
 
+function anyReasonableAddress() returns address {
+    address alice;
+    havoc alice;
```

Comment:
See discussion here: https://github.com/trusttoken/smart-contracts/pull/1108#discussion_r743292372.

I'm not sure whether the `havoc alice;` line is necessary.